src/sec_edgar_bulker/cli/main.py

```
@@ -0,0 +1,257 @@
+        # Setup logging
+        try:
+            logger = setup_logging(config.logging)
+        except Exception as e:
+            raise click.UsageError(f"Failed to setup logging: {str(e)}")
+            
+        # Run downloader
+        try:
+            asyncio.run(_run_downloader(config))
+            success = True  # Mark success before exiting
+        except Exception as e:
+            logger.error(f"Download failed: {str(e)}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)